project/Vagrantfile

```diff
@@ -34,14 +34,24 @@ end
 puts "ml-version=#{@ml_version}"
 puts "nr-hosts=#{@nr_hosts}"
 
+@priv_net_ip = ENV['PRIV_NET_IP'] || @properties['PRIV_NET_IP'] || "192.168.33.10"
+puts "priv-net-ip=#{@priv_net_ip}"
+
+
+@shared_folder_host = ENV['SHARED_FOLDER_HOST'] || @properties['SHARED_FOLDER_HOST'] || "./"
```

Comment:
./ = Vagrantfile folder, that is already mounted as /vagrant. I'd suggest no default for this..